src/Fantomas.Tests/ComputationExpressionTests.fs

```diff
@@ -2436,3 +2436,42 @@ async {
     return model.Prop
 }
 """
+
+[<Test>]
+let ``Idempotency problem when using lower MaxLineLength than default, 2176`` () =
```

Comment:
This problem is not related to computation expressions, please move to `DotIndexedGetTests`.
The test name should not start with an uppercase letter. See https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#unit-test

The test name itself does also not cover what the problem case is.
Rename to `multiline parentheses objectExpression in indexer expression, 2176`

Reply:
Done

Reply:
Thanks.